packages/simplebar-vue/index.vue

```diff
@@ -10,8 +10,8 @@
             class="simplebar-content-wrapper"
             ref="scrollElement"
             v-on="{
```

Comment:
For Vue 3 versions it is required to use `v-bind`, not `v-on` for such a construction. Check the [docs](https://v3.vuejs.org/guide/migration/listeners-removed.html#listeners-removed) plese